doc/conf.py

```diff
@@ -48,11 +49,13 @@
 # built documents.
 #
 # The short X.Y version.
-execfile(os.path.join(os.path.dirname(__file__), '../happybase/_version.py'))
-version = __version__
+
+version_path = join(dirname(__file__), '../happybase/_version.py')
+with open(version_path) as fp:
+    version = imp.load_source("_version", version_path, fp).__version__
```

Comment:
`exec()` seems like overkill.  If I had my way, I would likely put the version (and any other metadata about the module) into a separate file (json, ini, etc.), and read the data from there.